cpp/tests/binaryop/binop-integration-test.cpp

```diff
@@ -44,7 +44,24 @@ TEST_F(BinaryOperationIntegrationTest, Add_Scalar_Vector_SI32_FP32_SI64) {
   ASSERT_BINOP<TypeOut, TypeLhs, TypeRhs>(*out, lhs, rhs, ADD());
 }
 
-TEST_F(BinaryOperationIntegrationTest, Sub_Scalar_Vector_SI32_FP32_SI64) {
+TEST_F(BinaryOperationIntegrationTest, Add_Vector_Vector_SI32_FP32_FP32) {
```

Comment:
Just a question: why is this set of tests called "integrationTest"? Is it really an integration test? They look like unit tests to me.

Reply:
Not entirely sure.  I think this is probably pretty old code.